datafusion/physical-expr/Cargo.toml

```diff
@@ -74,7 +74,6 @@ uuid = { version = "^1.2", features = ["v4"] }
 
 [dev-dependencies]
 criterion = "0.5"
-datafusion = { workspace = true }
```

Comment:
Hm, why we added this dependency. Looks not correct. 🤔

Reply:
It looks like it was added to allow for some example code in rustdoc.

Reply:
Sorry I missed this on initial review.